backend/database/migrations/4_create_users_table.ts

```diff
@@ -6,9 +6,12 @@ export default class extends BaseSchema {
   async up() {
     this.schema.createTable(this.tableName, (table) => {
       table.increments('id').notNullable()
-      table.string('full_name').nullable()
-      table.string('email', 254).notNullable().unique()
-      table.string('password').notNullable()
+      table.string('usos_id').notNullable().unique()
+      table.string('first_name').notNullable()
+      table.string('last_name').notNullable()
+      table.string('student_number').notNullable().unique()
+      table.string('access_token').nullable()
```

Comment:
CZEMU MY CHCEMY TO TRZYMAĆ W BAZIE DANYCH 😭 TO JEST ZŁY POMYSŁ CHŁOPAKI

Reply:
+++, jak pisalem wyzej, wywal to z bazy wgl

Reply:
usos_id bym zostawil, zebysmy mogli jakkolwiek mieć możliwość "rejestracji" użytkownika

Reply:
@olekszczepanowski tak tak tak, tylko tokeny (accesstoken, accesssecret)